meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -30,19 +44,65 @@ interface IShowStyleVariantsProps {
 interface IShowStyleVariantsSettingsState {
 	editedMappings: ProtectedString<any>[]
 	timestampedFileKey: number
+	dndVariants: ShowStyleVariant[]
 }
 
+interface DragVariant {
+	index: number
+	type: ShowStyleDragDropTypes
+}
+
+const SET_STATE_DELAY: number = 50
```

Comment:
The type annotation here is irrelevant, since it is constant with a specified literal.

Reply:
Changed to use the suggestion!